spec/swagger_schemas/models/participant.rb

```diff
@@ -92,308 +90,234 @@
-                  nullable: false,
-                  example: Course::IDENTIFIERS.first,
-                  enum: Course::IDENTIFIERS,
-                },
-                schedule_identifier: {
-                  description: "The new schedule of the participant",
-                  nullable: true,
-                  type: :string,
-                  example: Schedule::IDENTIFIERS.first,
-                  enum: Schedule::IDENTIFIERS,
-                },
-                cohort: {
-                  description: "Indicates which call-off contract would fund this participant's training. 2021 indicates a participant that has started, or will start, their training in the 2021/22 academic year. Once a provider accepts an application, they may change a participant's cohort up until the point of submitting a started declaration.",
-                  type: :string,
-                  nullable: true,
-                  example: "2022",
-                },
-                npq_application_id: {
-                  description: "The ID of the NPQ application that was accepted to create this enrolment",
-                  type: :string,
-                  format: :uuid,
-                  nullable: false,
-                  example: "db3a7848-7308-4879-942a-c4a70ced400a",
-                },
-                eligible_for_funding: {
-                  description: "Indicates whether this NPQ participant would be eligible for funding from the DfE",
-                  type: :boolean,
-                  nullable: false,
-                  example: true,
-                },
-                training_status: {
-                  description: "The training status of the NPQ participant",
-                  type: :string,
-                  enum: Application.training_statuses.keys,
-                  example: Application.training_statuses.keys.first,
-                },
-                school_urn: {
-                  description: "The Unique Reference Number (URN) of the school where this NPQ participant is employed",
-                  type: :string,
-                  nullable: true,
-                  example: "106286",
-                },
-                targeted_delivery_funding_eligibility: {
-                  description: "Whether or not this application is eligible for Targeted Delivery Funding uplift",
-                  nullable: false,
-                  type: :boolean,
-                  example: true,
-                },
-                funded_place: {
-                  description: "Indicates whether or not this participant’s training is being funded by DfE",
-                  nullable: true,
-                  type: :boolean,
-                  example: true,
-                },
-              },
-            },
-          },
+}.tap { |h|
+  h[:v2] = h[:v1].deep_dup
+  h[:v2][:properties][:attributes][:properties].except!(:participant_id, :npq_courses, :funded_places)
+  h[:v2][:properties][:attributes][:properties][:npq_enrolments] = {
```

Comment:
Instead of replacing this could we just modify it to add the extra attributes in v3? something like:

```
h[:v3][:properties][:attributes][:properties][:npq_enrolments][:items][:properties].merge!({
    email: {
      description: "The email address registered for this NPQ participant",
      type: :string,
      nullable: false,
      example: "isabelle.macdonald2@some-school.example.com",
    },
    withdrawal: {
      description: "The details of an NPQ Participant withdrawal",
      type: :object,
      nullable: true,
      required: %i[reason date],
      example: nil,
      properties: {
        reason: {
          description: "The reason a participant was withdrawn",
          type: :string,
          nullable: false,
          example: Participants::Withdraw::WITHDRAWAL_REASONS.first,
          enum: Participants::Withdraw::WITHDRAWAL_REASONS,
        },
        date: {
          description: "The date and time the participant was withdrawn",
          type: :string,
          nullable: false,
          format: :"date-time",
          example: "2021-05-31T02:22:32.000Z",
        },
      },
    },
    deferral: {
      description: "The details of an NPQ Participant deferral",
      type: :object,
      nullable: true,
      required: %i[reason date],
      example: nil,
      properties: {
        reason: {
          description: "The reason a participant was deferred",
          type: :string,
          nullable: false,
          example: Participants::Defer::DEFERRAL_REASONS.first,
          enum: Participants::Defer::DEFERRAL_REASONS,
        },
        date: {
          description: "The date and time the participant was deferred",
          type: :string,
          nullable: false,
          format: :"date-time",
          example: "2021-05-31T02:22:32.000Z",
        },
      },
    },
    created_at: {
      description: "The date the application was created",
      type: :string,
      nullable: false,
      format: :"date-time",
      example: "2021-05-31T02:21:32.000Z",
    },
  })
```